BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeHttpClient.kt

```diff
@@ -18,164 +16,99 @@ internal class BraintreeHttpClient(
 ) {
 
     /**
-     * Make a HTTP GET request to Braintree using the base url, path and authorization provided.
-     * If the path is a full url, it will be used instead of the previously provided url.
-     * @param path The path or url to request from the server via GET
-     * @param configuration configuration for the Braintree Android SDK.
-     * @param authorization
-     * @param callback [NetworkResponseCallback]
+     * Make an asynchronous Braintree authenticated HTTP request.
+     *
+     * @param request the braintree http request.
```

Comment:
Update since this needs to handle requests being built for BT and PP APIs.
```suggestion
     * @param request the http request.
```

Reply:
I think this still holds for `BraintreeHttpClient`. For PP requests we should probably use a different HttpClient that doesn't append authentication headers to the request.

Reply:
I'll commit this one as well.